README.md

```diff
@@ -1,51 +1,127 @@
+
+With Wing, we believe we can provide a very elegant solution for both.
+As seen below, Wing has phase modifiers that are understood by the language server, which allows it to suggest different completion options for the same object, based on the execution phase from which is it accessed.
+[![Same object - different interfaces](./logo/diff-interfaces.gif)](https://youtu.be/y7OGRC2f5gk)
+
+As can be seen in the below section, creating a new language that is tailored to the cloud from the ground up also allows us to assemble a variety of features (some of which exist in other languages, of course) that, when put together, offer a delightful cloud development experience.
+
+## Main features that make Wing tailored for cloud development
+1. [Cloud services](https://docs.winglang.io/concepts/resources) are first-class citizens.
+    - They have [phase modifiers](https://docs.winglang.io/reference/spec#13-phase-modifiers) with different interfaces for config or runtime (preflight and inflight).
+2. Higher level of cloud abstraction.
+    - [Standard library](https://docs.winglang.io/reference/wingsdk-spec) that abstracts main cloud services.
+    - Built-in dependency injection system that resolves services to specific clouds at compile time.
+3. [Distributed computing primitives](https://docs.winglang.io/concepts/inflights).
+    - [Default immutability](https://docs.winglang.io/blog/2023/02/02/good-cognitive-friction#immutable-by-default).
+    - [Implicit async](https://docs.winglang.io/reference/spec#113-asynchronous-model), explicit defer.
+4. [Compiler plugins](https://docs.winglang.io/reference/compiler-plugins) that can be used to customize the compilation output, such as infrastructure definitions.
+5. [JavaScript interoperability](https://docs.winglang.io/reference/spec#5-interoperability).
+6. Automatic generation of IAM policies and other cloud mechanics based on intent.
+7. [Native JSON support](https://docs.winglang.io/reference/spec#114-json-type).
+8. More in the [spec](https://docs.winglang.io/reference/spec).
+
+## What does the development flow look like with Wing
+1. Write code that targets an abstract cloud.
+2. Visualize, test, and interact with it using the Wing console and its instant hot reloading.
+3. Compile to JS and Terraform, targeting the desired cloud and provisioning engine.
+4. Deploy and manage with your existing tools.
+
+[![Short demo](./logo/demo.gif)](https://youtu.be/x0xfIoY5y6E)
+
+## Is there a company behind Wing / what is the business model
+Monada Inc. is leading the development of Winglang.
```

Comment:
```suggestion
[Monada](https://monada.co) is leading the development of Winglang.
```

idk if there is a legal reason to include "inc" but I would avoid if possible, it's a bad vibe

Reply:
I think it can be removed.
The reason I didn't add a link to monada's website is because it will soon redirect to winglang's website, but I guess we can figure out what to do with it then..